src/VsixTesting.Installer/VsixTesting.Installer.csproj

```diff
@@ -16,5 +24,9 @@
       <SpecificVersion>false</SpecificVersion>
       <Private>False</Private>
     </Reference>
+    <PackageReference Include="Nerdbank.GitVersioning" Version="2.1.65" PrivateAssets="all" />
+    <PackageReference Include="EnvDTE" Version="8.0.0" />
```

Comment:
❓ Why no `PrivateAssets="all"` on this one?

Reply:
Good catch!